tests/dashbio_demos/dash-pileup/app.py

```diff
@@ -0,0 +1,200 @@
+import os
+import dash_html_components as html
+import dash_core_components as dcc
+from dash.dependencies import Input, Output
+import dash_bio
+import pandas as pd
+import dash_table
+from dash_table.Format import Format
+import numpy as np
+
+try:
+    from layout_helper import run_standalone_app
+except ModuleNotFoundError:
+    from .layout_helper import run_standalone_app
```

Comment:
```suggestion
from layout_helper import run_standalone_app
```
This is part of some of the changes I'm making to demo apps for #518, it would make sense to have this app follow the updated form before merging.